spec/jekyll_seo_tag_spec.rb

```diff
@@ -124,7 +124,6 @@
 
       it 'outputs the logo' do
         expect(json_data['logo']).to eql('http://example.invalid/logo.png')
-        expect(json_data['url']).to eql('http://example.invalid')
```

Comment:
`url` [should be the URL of the current page, not the site](https://schema.org/BlogPosting). Since it doesn't have anything to do with outputting the logo anyway, I just removed this check.